### PR TITLE
added start script in package.json

### PR DESCRIPTION
**Pull request by Nahdus**
Added start command under script in package.json

user can run 

`npm start`

instead of 

node app.js 300

mitigates issue #1

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -436,4 +436,4 @@
       "integrity": "sha1-IpnwLG3tMNSllhsLn3RSShj2NPw="
     }
   }
-}
+}
```
Comment (maanavshah) [inline package-lock.json:439]: Please revert this file change

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,354 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (maanavshah) [inline yarn.lock:1]: why this file is required?

Comment (Nahdus) [reply yarn.lock:1]: that is a lock file generated by yarn (package manager) install.
Its similar to a package-lock.json file generated when one uses npm install
It is meant to lock the exact version of the package/modules used during development.

https://robertcooper.me/post/how-yarn-lock-files-work-and-upgrading-dependencies

when these lock files are not present the package.json file typically has ^ sign prefixed to version number of each dependency, which locks only the major number of the version.

eg 
`"react": "^17.0.2"`,
accepts:
`"react": "^17.0.2"`,
`"react": "^17.0.2"`,
`"react": "^17.0.2"`,

**Comment by Nahdus** [comment]
Hi, @maanavshah removed the yarn.lock file and reverted package-lock.json changes

**Review by maanavshah — APPROVED** [review]

**Pull request merged**